exdreader.py

```diff
@@ -268,7 +267,7 @@ def __str__(self):
 
 
 class ExcelListFile:
-    def __init__(self, data: list[bytes]):
+    def __init__(self, data: List[bytes]):
```

Comment:
Why only change this one case when there is multiple other using the same syntax before the change? 
And I see no reason for this change

Reply:
If there's more, my IDE didn't complain about it. However, you absolutely should change this because of `list` as a valid type only showing up in python 3.9. See [docs](https://docs.python.org/3/library/typing.html#typing.List):

> *Deprecated since version 3.9*: [builtins.list](https://docs.python.org/3/library/stdtypes.html#list) now supports subscripting (`[]`). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).